crates/astria-conductor/src/executor/mod.rs

```diff
@@ -214,6 +231,37 @@ impl<C: ExecutionClient> Executor<C> {
             return Ok(Some(execution_hash.clone()));
         }
 
+        let mut final_response = Ok(Some(self.execution_state.clone()));
+        // TODO: (GHI 250 - https://github.com/astriaorg/astria/issues/250):
+        // if blocks are coming from DA, do we still need them to pass through
+        // the queue? if not, we can skip the insert and pop_blocks calls
```

Comment:
yeah blocks from DA should still go through the queue as there could be gaps/out of order